common/data_refinery_common/job_management.py

```diff
@@ -147,75 +147,15 @@ def create_downloader_job(undownloaded_files: List[OriginalFile],
 
     return True
 
+
 def create_processor_jobs_for_original_files(original_files: List[OriginalFile],
                                              downloader_job: DownloaderJob=None,
                                              volume_index: str=None):
     """
     Creates one processor job for each original file given.
     """
     for original_file in original_files:
-        sample_object = original_file.samples.first()
-
-        if original_file.is_blacklisted():
-            logger.debug("Original file had a blacklisted extension of %s, skipping",
-                         extension=original_file.get_extension(),
-                         original_file=original_file.id)
-            original_file.delete_local_file()
-
-        # Fix for: https://github.com/AlexsLemonade/refinebio/issues/968
-        # Basically, we incorrectly detected technology/manufacturers
-        # for many Affymetrix samples and this is a good place to fix
-        # some of them.
-        if original_file.is_affy_data():
-            # Only Affymetrix Microarrays produce .CEL files
-            sample_object.technology = 'MICROARRAY'
-            sample_object.manufacturer = 'AFFYMETRIX'
-            sample_object.save()
```

Comment:
This code block isn't present in the other function that you're calling. I think we still want to be doing this right?

Reply:
Looks like we still have `2719` samples affected by this. If the original bug is fixed, can we update them directly and remove this code from here?

```
>>> Sample.objects.exclude(technology='MICROARRAY', manufacturer='AFFYMETRIX').filter(Q(original_files__source_filename__iendswith='.cel') | Q(original_files__source_filename__iendswith='.cel.gz')).distinct().count()
2719
```

Reply:
Yeah that sounds good to me!

Reply:
ok I ran the following code and corrected them:

```
>>> from data_refinery_common.models import Sample
>>> from django.db.models.expressions import F, Q
>>> affected_samples = Sample.objects.exclude(technology='MICROARRAY', manufacturer='AFFYMETRIX').filter(Q(original_files__source_filename__iendswith='.cel') | Q(original_files__source_filename__iendswith='.cel.gz')).distinct()
>>> affected_samples.count()
2719
>>> affected_samples.update(technology='MICROARRAY', manufacturer='AFFYMETRIX')
2719
```